### PR TITLE
[DEV APPROVED] 7413 - Updating favicon paths

### PR DESCRIPTION
**Pull request by johnplayer1982**
# 7413 - Updating favicon paths

The blog favicon's are not being found since the blog was moved to ``https://www.moneyadviceservice.org.uk/blog``, this is because in the blog project the favicons are still being referenced as if they are in the root of the domain, eg: ``href="/favicon-128.png"``.

Since the domain has changed, the favicons are really in ``https://www.moneyadviceservice.org.uk/blog``.  This change adds a partial to hold the favicons and updates the paths to point to ``/blog/``

Below you can see where the blog is looking for the favicons currently, and a demo of where the blog will find the after this change

|  https://www.moneyadviceservice.org.uk/apple-touch-icon-114x114.png  |  https://www.moneyadviceservice.org.uk/blog/apple-touch-icon-114x114.png |
|-------|--------|
|screen shot 2016-06-09 at 12 27 55|screen shot 2016-06-09 at 12 28 03

### CODE REVIEW

**Comment by slaywell** [comment]
👍 - great to get all the sizes / formats in there

File: app/views/shared/_favicons.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<link rel="apple-touch-icon-precomposed" sizes="57x57" href="/blog/apple-touch-icon-57x57.png" />
```
Comment (jongilbraith) [inline app/views/shared/_favicons.html.erb]: Might be good with these to look for `ENV['RAILS_RELATIVE_URL_ROOT']` so we don't have a hard coded dependency to where we're serving it.

Something like:

```
<link rel="apple-touch-icon-precomposed" sizes="57x57" href="<%= ENV['RAILS_RELATIVE_URL_ROOT'] %>/apple-touch-icon-57x57.png" />
```

**Comment by jongilbraith** [comment]
Other than my comment, all the rest looks 👍

**Comment by pmcostello** [comment]
I second Jon's comment above, using the relative_url_root would be preferable, other than that all looks good 👍

**Comment by johnplayer1982** [comment]
Thanks for review, what about the paths in ``public/code.txt``, its just a text file so I dont think I can use any ruby code

**Comment by jongilbraith** [comment]
I've no idea what the purpose of that file could be @johnplayer1982. I'd hazard a guess it might have been a temporary note of Yomi's that he accidentally committed.

I reckon just remove it.

**Comment by johnplayer1982** [comment]
@jongilbraith done, gone!

**Comment by jongilbraith** [comment]
@johnplayer1982 excellent! 👍

**Pull request merged**